### PR TITLE
Full review of repository code by Daria Hrychyna

### PR DESCRIPTION
**Pull request by DariaHrychyna**

### CODE REVIEW

File: netflix-clone-yt-master/src/app/shared/components/movie-carousel/movie-carousel.component.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  @Input() videoContents: IVideoContent[] = [];
+  @Input() title!: string;
+  @ViewChild('swiperContainer') swiperContainer!: ElementRef;
+  selectedContent: string | null = null;
+  constructor() { }
+  ngAfterViewInit(): void {
+   this.initSwiper();
+  }
+
+  ngOnInit() {
```
Comment (DariaHrychyna) [inline netflix-clone-yt-master/src/app/shared/components/movie-carousel/movie-carousel.component.ts:34]: Consider initializing any logic or variables here if needed.

File: netflix-clone-yt-master/src/app/shared/components/movie-carousel/movie-carousel.component.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        1800: {
+          slidesPerView: 5,
+          slidesPerGroup: 6,
+          spaceBetween: 5,
+          centeredSlides: false,
+        }
+      }
+    })
+  }
+
```
Comment (DariaHrychyna) [inline netflix-clone-yt-master/src/app/shared/components/movie-carousel/movie-carousel.component.ts:77]: Ensure to provide appropriate type annotation for the movie parameter.

File: netflix-clone-yt-master/src/app/app.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  beforeEach(() => TestBed.configureTestingModule({
+    imports: [AppComponent]
+  }));
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
+    expect(app).toBeTruthy();
+  });
+
```
Comment (DariaHrychyna) [inline netflix-clone-yt-master/src/app/app.component.spec.ts:14]: Use template literals consistently, or avoid them for simple strings.

File: netflix-clone-yt-master/src/app/app.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  it(`should have the 'netflix-clone-yt' title`, () => {
+    const fixture = TestBed.createComponent(AppComponent);
+    const app = fixture.componentInstance;
```
Comment (DariaHrychyna) [inline netflix-clone-yt-master/src/app/app.component.spec.ts:17]: Consider checking for the existence of the 'title' property before comparing its value.

File: netflix-clone-yt-master/src/app/shared/components/movie-carousel/movie-carousel.component.ts (hunk 1 above)
Comment (omruska) [reply netflix-clone-yt-master/src/app/shared/components/movie-carousel/movie-carousel.component.ts:34]: ok👍

File: netflix-clone-yt-master/src/app/shared/components/movie-carousel/movie-carousel.component.ts (hunk 2 above)
Comment (omruska) [reply netflix-clone-yt-master/src/app/shared/components/movie-carousel/movie-carousel.component.ts:77]: thanks. do this later👍

File: netflix-clone-yt-master/src/app/app.component.spec.ts (hunk 3 above)
Comment (omruska) [reply netflix-clone-yt-master/src/app/app.component.spec.ts:14]: yes👍

File: netflix-clone-yt-master/src/app/app.component.spec.ts (hunk 4 above)
Comment (omruska) [reply netflix-clone-yt-master/src/app/app.component.spec.ts:17]: ok👍

**Pull request merged**